test/TestRuleClass.py

```diff
@@ -1,3 +1,6 @@
+from ansiblelint.rules import AnsibleLintRule
```

Comment:
Why did you rename it?

Reply:
File contains tests related to base rule class, previous name was referring only to attributes.